### PR TITLE
Add router doc #91.

### PR DESCRIPTION
**Pull request by sjstebbins**

### CODE REVIEW

File: docs/TuxRouter.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#Tux/Router
+
+## <a id="Premise"></a>Premise [#](#Premise)
+>The Tux Router simply exposes the functionality of the amazing **react-router** npm and allows it to be easily accessible in your Tux projects. This documentation is borrowed from the **react-router** docs and is only slightly adjusted. To read more about the fabulous **react-router** visit the [**react-router** github page.](https://github.com/rackt/react-router)
+
```
Comment (drabinowitz) [inline docs/TuxRouter.md]: Need to mention how `tux` breaks up the `react-router` into individual modules.  Thus, in `tux` you require the `tux/Router/ROUTER COMPONENT`.  Maybe we can have links to each corresponding component within the `react-router`.

File: docs/TuxRouter.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#Tux/Router
+
+##Table of Contents
```
Comment (Cheerazar) [inline docs/TuxRouter.md]: Add space between # and T

File: docs/TuxRouter.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#Tux/Router
```
Comment (Cheerazar) [inline docs/TuxRouter.md]: Add space between # and T.

File: docs/TuxRouter.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+##Table of Contents
+<ol>
+  <li><a href="#Premise">Premise</a></li>
+  <li><a href="#Implementation">Implementation</a></li>
+</ol>
+
+## <a id="Premise"></a>Premise [#](#Premise)
+>The Tux Router simply exposes the functionality of the amazing **react-router** npm and allows it to be easily accessible in your Tux projects. This documentation is borrowed from the **react-router** docs and is only slightly adjusted. To read more about the fabulous **react-router** visit the [**react-router** github page.](https://github.com/rackt/react-router)
```
Comment (Cheerazar) [inline docs/TuxRouter.md]: Instead of using the \*\* use the \` to surround react-router.

File: docs/TuxRouter.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+## <a id="Implementation"></a>Implementation [#](#Implementation)
+
+To use the react-router in Tux, simply require it and expose the features you need. In this case: Route and  RouterHandler.
```
Comment (Cheerazar) [inline docs/TuxRouter.md]: Surround react-router with \`.

**Pull request merged**